lib/transition.js

```diff
@@ -30,9 +41,12 @@ class Transition {
         if (prop[0] === '-') continue
 
         let prefixer = this.prefixes.add[prop]
+        let inSpecific = this.inVendorSpec(decl)
```

Comment:
If we will merge `declPrefixes` and `inSpecific` generation we will be able to avoid the double call of `inVendorSpec`

Reply:
I completely remove ```process``` method cause now it seems to be useless at all.